compiler/src/iree/compiler/Codegen/LLVMCPU/KernelDispatch.cpp

```diff
@@ -2950,12 +2951,18 @@ setLoweringConfigForComputeOps(mlir::FunctionOpInterface entryPointFn,
       TilingConfig::create(rootLoweringConfig);
   SmallVector<int64_t> distTileSizes, parallelVecTileSizes;
   SmallVector<bool> distScalableTileSizes, parallelVecScalableTileSizes;
-  if (tilingConfig->getNumTilingLevels() > 0) {
-    distTileSizes = tilingConfig->getDistributionTileSizes();
-  }
-  if (tilingConfig->getNumTilingLevels() > 1) {
-    std::tie(parallelVecTileSizes, parallelVecScalableTileSizes) =
-        tilingConfig->getVectorCommonParallelSizes();
+  if (rootLoweringConfig.hasTilingLevel(TilingLevel::DistributionTiles)) {
+    distTileSizes = rootLoweringConfig.getStaticTilingLevelSizes(
+        TilingLevel::DistributionTiles, rootOperation);
+  }
```

Comment:
According to the comment in `getNewLoweringConfig`:
> // Distribution tile sizes is a must for rootOp.

So, wouldn't this be always true:
```cpp
if (rootLoweringConfig.hasTilingLevel(TilingLevel::DistributionTiles))
```

Reply:
Good catch, let's convert it to an assertion.